README.md

```diff
@@ -70,7 +78,42 @@
 - **Use of Flexbox in css**
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+## 🚀 Live Demo <a name="live-demo"></a>
 
+- [Live Demo Link](index.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites <a name="prerequisites"></a>
+
+In order to run this project you need to create a repository.
+
+### Setup <a name="about-project"></a>
+
+Create a .github/workflows folder and add a copy of .github/workflows/linters.yml to that folder.
+
+### Install <a name="install"></a>
+
+Run npm install --save-dev hint@7.x
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+### Usage <a name="usage"></a>
+
+To catch programming errors,stylistic errors,bugs and suspicious constructs.
+
+### Run tests <a name="run-tests"></a>
+
+npx stylelint "**/*.{css,scss}"
+
+### Deployment <a name="deployment"></a>
+
+You can deploy this project using Chrome Browser
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Good job implementing this section requested by the previous reviewer 👍🏼 However, if I follow these steps, I won't be able to get a local copy up and running on my machine, these instructions are to run linters.

Please consider updating them! 🤓 🛠️ 

💡 _Hint: You could mention prerequisites as trivial as having a code editor and a browser. For the setup, you could ask to clone your GitHub repo inside a local folder. For installation, you could ask to run `npm install` to install project dependencies. For usage, you could ask to open the `index.html` file on the browser, and so on... At the moment you haven't worked on any tests or deployment, so you can skip those sections for now._